reconcile/test/test_quay_repos_api.py

```
@@ -0,0 +1,118 @@
+) -> _TestableIntegration:
+    return _TestableIntegration(
+        QuayReposIntegrationParams(org_name=org_name, instance_name=instance_name)
+    )
+
+
+def _make_org(
+    name: str = "myorg",
+    instance_name: str = "quay.io",
+    instance_url: str = "https://quay.io",
+    managed_repos: bool = True,
+    token_path: str = "secret/quay/myorg",
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to function default: "token_path"

(S107)